src/relay/op/annotation/annotation.cc

```diff
@@ -37,6 +38,31 @@ RELAY_REGISTER_OP("on_device")
 .set_attr<FInferCorrectLayout>("FInferCorrectLayout",
                                ElemwiseArbitraryLayout);
 
+Expr StopFusion(Expr data) {
+  static const Op& op = Op::Get("annotation.stop_fusion");
+  return CallNode::make(op, {data}, Attrs{}, {});
+}
+
+TVM_REGISTER_API("relay.op.annotation._make.stop_fusion")
+.set_body_typed<Expr(Expr)>([](Expr data) {
+    return StopFusion(data);
+});
+
+RELAY_REGISTER_OP("annotation.stop_fusion")
+.describe(R"code(Annotate an expression to prevent it being fused with previous expressions.)code"
+TVM_ADD_FILELINE)
+.set_num_inputs(1)
+.add_argument("data", "Tensor", "The input data.")
+.add_type_rel("Identity", IdentityRel)
+.set_support_level(10)
+.set_attr<TOpPattern>("TOpPattern", kOpaque)
+.set_attr<TOpIsStateful>("TOpIsStateful", false)
+.set_attr<FInferCorrectLayout>("FInferCorrectLayout", ElemwiseArbitraryLayout)
+.set_attr<FTVMCompute>("FTVMCompute",
```

Comment:
do we still need the compute? give that stop_fusion simply get transformed away.

Reply:
FTVMCompute is needed to enable constant folding

Reply:
OK